cli/root.go

```diff
@@ -0,0 +1,78 @@
+package cli
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/prometheus/alertmanager/cli/format"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var cfgFile string
+
+// RootCmd represents the base command when called without any subcommands
+var RootCmd = &cobra.Command{
+	Use:   "amtool",
+	Short: "Alertmanager CLI",
+	Long: `View and modify the current Alertmanager state.
+
+[Config File]
+
+The alertmanger tool will read a config file from $HOME/.amtool.yml or /etc/amtool.yml the options are as follows
```

Comment:
Would probably be better to read the path from `env AMTOOL_CONFIG` and default to `$HOME/.amtool.yml`.
Not sure whether we need `/etc/amtool.yml` to be generally considered but that could be the fallthrough option.